src/applicationcomposer/messageHandlers/generateResourceHandler.ts

```diff
@@ -133,21 +132,17 @@ async function generateResource(prompt: string) {
 
         const elapsedTime = globals.clock.Date.now() - startTime
 
-        // TODO-STARLING: Reduce/remove below
-        getLogger().debug(`===== CW Chat prompt start ====`)
-        getLogger().debug(prompt)
-        getLogger().debug(`===== CW Chat prompt end ======`)
-        getLogger().debug(`===== CW Chat metadata start ======`)
-        getLogger().debug(`CW Chat conversationId = ${conversationId}`)
-        getLogger().debug(`CW Chat metadata = \n${JSON.stringify(metadata, undefined, 2)}`)
-        getLogger().debug(`CW Chat supplementaryWebLinks = \n${JSON.stringify(supplementaryWebLinks, undefined, 2)}`)
-        getLogger().debug(`CW Chat references = \n${JSON.stringify(references, undefined, 2)}`)
-        getLogger().debug(`===== CW Chat metadata end ======`)
-        getLogger().debug(`===== CW Chat raw response start ======`)
-        getLogger().debug(`${response}`)
-        getLogger().debug(`===== CW Chat raw response end ======`)
-        getLogger().debug(`CW Chat initial response time = ${initialResponseTime} ms`)
-        getLogger().debug(`CW Chat elapsed time = ${elapsedTime} ms`)
+        getLogger().debug(
+            `CW Chat Debug message:
+             prompt = "${prompt}",
+             conversationId = ${conversationId},
+             metadata = \n${JSON.stringify(metadata, undefined, 2)},
+             supplementaryWebLinks = \n${JSON.stringify(supplementaryWebLinks, undefined, 2)},
+             references = \n${JSON.stringify(references, undefined, 2)},
+             response = "${response}",
+             initialResponse = ${initialResponseTime} ms,
+             elapsed time = ${elapsedTime} ms`
```

Comment:
p.s. you could use `indent()` to ensure consistent indent https://github.com/aws/aws-toolkit-vscode/blob/961ff52e05ed45d25c749a2237de6b2b3237f076/src/shared/utilities/textUtilities.ts#L39